DataFormats/L1TrackTrigger/interface/TTStub.h

```diff
@@ -80,6 +88,8 @@ TTStub< T >::TTStub()
   theDetId = 0;
   theDisplacement = 999999;
   theOffset = 0;
+  theRealOffset = 0;
+  theHardwareBend = 999999;
```

Comment:
Since the "999999" value is used in multiple places, it should be a const member of the class (e.g. `defaultHardwareBend`)

Reply:
Done